pkg/valueset/postgresql.go

```diff
@@ -0,0 +1,68 @@
+package valueset
+
+import (
+	"database/sql"
+	"net/url"
+)
+
```

Comment:
Move Config to pkg/valueset
Move everything else to pkg/valueset/pgx

Make a factory in pkg/Valueset

map[string]func(interface{}) (ValusetSetDB, error)

Config:
```
Config struct {
    []ValuesetDB
}
```
Unmarshal cponfig:
```
unmarshalConfig struct {
   Databases  map[string]interface{}
}
```
Use factory to unmarshal the config

ValuesetDB interface {
  ValueSetLookup(tableId, map[string]string) map[string]string,error
   GetTableIDs()[]
}
Use YAMLToMap under lsa/layers to convert yaml to map[string]interface{}